src/ostree/ot-builtin-admin.c

```diff
@@ -121,11 +123,6 @@ ostree_builtin_admin (int argc, char **argv, OstreeCommandInvocation *invocation
             }
         }
 
-      else if (g_str_equal (argv[in], "--"))
```

Comment:
Can you motivate this change?

Reply:
We need to support command patterns like this:
```console
$ ostree admin nsenter -- sh -c "echo hello"
```

In this command, if `--` is omitted, `-c` is processed by ostree and it fails.

So, nsenter need to support `--` processing. If `--` is processed here, nsenter cannot get the arguments after `--`.

Maybe deleting the logic here may break things. We need to add a OSTREE_BUILTIN_FLAG_ to specify whether a command needs to process `--` by itself. I'll do that later.